src/game/client.cpp

```diff
@@ -477,11 +477,15 @@ namespace client
                 items.emplace_back(v >= W_OFFSET && v < W_ITEM ? v : 0);
             }
         }
-        game::player1.loadweap.shrink(0);
-        loopv(items) if(game::player1.loadweap.find(items[i]) < 0)
-        {
-            game::player1.loadweap.add(items[i]);
-            if(game::player1.loadweap.length() >= W_LOADOUT) break;
+        game::player1.loadweap.clear();
+        for (const auto i : items) {
+            if (std::find(game::player1.loadweap.begin(),
+                game::player1.loadweap.end(), i) == game::player1.loadweap.end()) // NOT found
```

Comment:
I think the comment is not needed, even less experienced C++ programmers should know this idiom.

Also I'm pretty much the opposite of a fan of inline comments. They just increase the line length for no apparent reason and are hard to spot/find/read as well.